modules/kit-elements-defaults/assets/js/editor/utils.js

```diff
@@ -0,0 +1,6 @@
+export function isPopulatedObject( obj ) {
+	return obj &&
+		'object' === typeof obj &&
+		! Array.isArray( obj ) &&
+		Object.keys( obj ).length > 0;
+}
```

Comment:
`_.isEmpty`?

Reply:
I prefer not to use `underscore` functions to allow us to remove it in the future + `isEmpty` also checks for strings and arrays, and I want to check that the var is an object and it has data inside.